ios/BT/API/Model/ExposureDetectionSummary.swift

```diff
@@ -9,19 +9,19 @@ class ExposureDetectionSummary: Object, Codable {
   @objc dynamic var sequenceNumberInDay: Int = 0
   @objc dynamic var startOfDateReceived: Int = 0
   @objc dynamic var timezoneOffset: Int = 0
-  @objc dynamic var matchedKeyCount: UInt64 = 0
-  dynamic var attenuationDuration: AttenuationDuration = AttenuationDuration()
-  @objc dynamic var maximumRiskScore: UInt8 = 0
+  @objc dynamic var matchedKeyCount: Int64 = 0
+  @objc dynamic var attenuationDuration: AttenuationDuration? = AttenuationDuration()
+  @objc dynamic var maximumRiskScore: Int8 = 0
   @objc dynamic var riskScoreSumFullRange: Double = 0.0
 
   init(_ enExposureDetectionSummary: ENExposureDetectionSummary) {
     timezoneOffset = TimeZone.current.secondsFromGMT()
-    attenuationDuration.low = Int(truncating: enExposureDetectionSummary.attenuationDurations[0])
-    attenuationDuration.medium = Int(truncating: enExposureDetectionSummary.attenuationDurations[1])
-    attenuationDuration.high = Int(truncating: enExposureDetectionSummary.attenuationDurations[2])
+    attenuationDuration?.low = Int(truncating: enExposureDetectionSummary.attenuationDurations[0])
```

Comment:
Ugh, so sorry - the swagger API does indeed say attenuationDuration but it should be plural: attenuationDurations. I think Matt might even have asked me about this. But, I guess attenuationDurations plural it is.
Could you please update this?